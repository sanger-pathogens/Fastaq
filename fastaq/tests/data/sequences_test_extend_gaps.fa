>1
AC
>2
ACGTACGT
>3
ACGTNACGT
>4
ACGTACGTNACGTACGT
