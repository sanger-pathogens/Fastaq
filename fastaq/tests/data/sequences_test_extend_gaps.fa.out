>2
GTAC
>4
GTACNNNNNGTAC
